src/main/Java/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
I think `/src/main/Java` is usually capitalized as `src/main/java` as part of a Maven convention. Not sure if it's a big deal, but especially if you switch it to a Maven build it'll probably be annoying to deal with